src/Main.java

```diff
@@ -1,5 +1,53 @@
 public class Main {
     public static void main(String[] args) {
 
+        System.out.println("Задача 1");
+        int[] sum = {22, 23, 55, 6, 85};
+        int salarySum = 0;
+        for (int i : sum) {
+            salarySum += i;
+        }
+        System.out.println("Сумма трат за месяц составила " + salarySum + " руб");
+
+        System.out.println("Задача 2");
+        int[] sum1 = {22, 23, 55, 6, 85};
+        int max = -1;
+        for (int i = 0; i < sum1.length; i++) {
+            if (sum1[i] > max) {
+                max = sum1[i];
+            }
+        }
+        System.out.println("Максимальная сумма трат за неделю составила "+ max +" руб");
+        int min= sum1[0];
+        for (int i = 0; i <sum1.length; i++) {
+            if (sum1[i]< min) {
+                min = sum1[i];
+            }
+        }
+        System.out.println("Минимальная сумма трат за неделю составила "+min+" руб");
+
+
+        System.out.println("Задача 3");
+        int[] arr = {22, 23, 55, 6, 85};
+        int summ = 0;
+        int averageValue = 0;
+        for (double element: arr){
+            summ+=element;
+            averageValue=summ/5;
```

Comment:
1) Чтобы посчитать среднее - надо сумму поделить на количество элементов. Сумма нужно считать циклом, а вот среднее уже вне его (количество же не меняется).
2) 5 - это хардкод. Размер массива надо брать из его свойств
 ```suggestion
            averageValue=summ/arr.length;
```